glotaran/builtin/models/doas/test/test_doas_model.py

```diff
@@ -208,7 +209,8 @@ def test_doas_model(suite):
     assert suite.model.valid(suite.parameter)
 
     dataset = suite.sim_model.dataset["dataset1"].fill(suite.sim_model, suite.wanted_parameter)
-    data = xr.DataArray(suite.axis["time"], coords=[("time", suite.axis["time"])])
+    # TODO: Actually use this value for something or delete it.
+    # data = xr.DataArray(suite.axis["time"], coords=[("time", suite.axis["time"])])
```

Comment:
odd indeed, it gets overwritten a few lines down

Reply:
Guess this is a refactoring artifact, due to linter not complaining.